src/Illuminate/Mail/Events/MessageSent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Mail\Events;
+
+use Swift_Message;
+
+class MessageSent
+{
+    /**
+     * The Swift message instance.
+     *
+     * @var \Swift_Message
+     */
+    public $message;
+
+    /**
+     * The Swift transport.
+     *
+     * @var string
+     */
+    public $transport;
+
+    /**
+     * The Swift send result - format depends on the transport.
```

Comment:
But you return the same result at the `Mailer::send()` method, so whats the difference between there and the Event? On both places you have to handle the result depending of the transport, right?

Otherwise I have to replace the Mailer and adapt the MailEvents in my project, to add the result.
This way it should work for me, but I think it could be also a benefit for others.
For example an extension handle the bounces from the MailService automaticly.

But it's your decision.. leave or remove?